src/core/client/stream/local/local.graphql

```diff
@@ -1,3 +1,6 @@
+# This extends ./src/core/client/framework/lib/relay/local.graphql
```

Comment:
Maybe we can look into imports in the future!

Reply:
Interesting! Wonder if this would cause a problem like loading it twice.

Reply:
Note: The import order is currently defined by how the react compiler args are provided.

Reply:
imports from `graphql-tools` https://www.graphql-tools.com/docs/schema-loading/ is non standard, and it seems to be a different use case. It's used to stich together a full schema to use by the server, while this is about client schema extensions.